config/action.d/pf.conf

```diff
@@ -16,7 +16,7 @@
 # we don't enable PF automatically; to enable run pfctl -e 
 # or add `pf_enable="YES"` to /etc/rc.conf (tested on FreeBSD)
 actionstart = echo "table <<tablename>-<name>> persist counters" | pfctl -f- 
-              echo "block proto <protocol> from <<tablename>-<name>> to any port <port>" | pfctl -f-
+              echo "block proto <protocol> from <<tablename>-<name>> to any<actiontype>" | pfctl -f-
```

Comment:
Of course we can do that, but I would not so "accurate" with possible evil customizations in actions, BUT not in filters:
- if action defined wrong, it does not work (and shits with errors in f2b.log);
- if filter defined wrong, it will never find some failures (and can easily be overlooked, because no errors directly)